core/merginapi.cpp

```diff
@@ -2102,6 +2105,9 @@ void MerginApi::pushStartReplyFinished()
   {
     QByteArray data = r->readAll();
     QString serverMsg = extractServerErrorMsg( data );
+    if ( r->error() == QNetworkReply::OperationCanceledError )
+      serverMsg = tr( "Synchronisation canceled" );
```

Comment:
maybe pull it somewhere to the top of the file as cons string so it is just 1 string to translate?